doc/macsec/MACsec_hld.md

```diff
@@ -213,6 +226,13 @@ key                         = PORT:name               ; Interface name
 ; field                     = value
 macsec                      = profile                 ; MACsec profile name. if this filed is empty or isn't existed,
                                                       ; the MACsec function is disable.
+pfc_encryption_mode         = "bypass" / "encrypt" / "strict_encrypt"
+                                                       ; The behavior to PFC frames.
+                                                       ; "bypass": react clear and encrypted PFC frames, send clear PFC frames.
+                                                       ; "encrypt": react clear and encrypted PFC frames, send encrypted PFC frames.
+                                                       ; "strict": only react encrypted PFC frames, send encrypted PFC frames.
+                                                       ; If the specified mode cannot be supported by the platform, record an error in the log file.
```

Comment:
Also need a single CLI to change the encryption mode on all macsec enabled ports

Reply:
> if the user tries to configure a mode that is unsupported by the platform, do we fallback to a default pfc_encryption_mode?

I think the SONiC device should support both three modes definitely, Because we use the ACL to differentiate the modes and gearbox or ASIC should both support ACL. The three modes are for adapting the capability of neighbor devices.

Reply:
> Also need a single CLI to change the encryption mode on all macsec enabled ports

Yes, good suggestion. I think I will implement this CLI in the CLI layer instead of database layer.